apis/swagger.yml

```diff
@@ -2131,6 +2115,27 @@ definitions:
           HostConfig:
             $ref: "#/definitions/HostConfig"
 
+  LogConfig:
```

Comment:
I am afraid we still need to update the routes of `  /containers/{id}/logs:` in this file, since the status code in the code from manager are more than that has been defined in `  /containers/{id}/logs:`. 
For example, 401.

Reply:
Make senses. Will do